Dockerfile-alpine.template

```diff
@@ -0,0 +1,28 @@
+# A GeoServer installation
+FROM tomcat:%%TOMCAT_VERSION%%-jre%%JRE_VERSION%%-alpine
+
+ENV GEOSERVER_VERSION  %%GEOSERVER_VERSION%%
+ENV GEOSERVER_WAR_URL  https://github.com/azavea/docker-geoserver/releases/download/0.0.0/geoserver-${GEOSERVER_VERSION}-war.zip
+ENV GEOSERVER_MONITORING_URL https://github.com/azavea/docker-geoserver/releases/download/0.0.0/geoserver-${GEOSERVER_VERSION}-monitor-plugin.zip
+ENV GEOSERVER_DIR  ${CATALINA_HOME}/webapps/geoserver
+ENV GEOSERVER_EXT_DIR  ${GEOSERVER_DIR}/WEB-INF/lib
+ENV GEOSERVER_DATA_DIR /data
+
+RUN cd /tmp \
```

Comment:
It might be good to reset `WORKDIR` after this instruction because otherwise I think you'll be in `/tmp`.

Reply:
Fixed in 9b88114.

Reply:
Fixed in 9b88114.